### PR TITLE
Add search bar

### PR DESCRIPTION
**Pull request by rottencandy**
Add:
- `SearchBar` component consisting of `form`, `TextInput` and `Button`.
- Accept callback to run on form submission
- Test cases
- Styles

### CODE REVIEW

**Comment by invincibleJai** [comment]
@rottencandy are all commits needed? if not squash in 1

File: src/Homepage.tsx [hunk 1]
```diff
@@ -18,7 +20,13 @@ const Header = (
 )
 
 const Homepage: React.FC = () => {
-  return <Page header={Header}></Page>
+  return (
```
Comment (invincibleJai) [inline src/Homepage.tsx]: can use implicit return

File: src/Homepage.tsx [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 
 import '@patternfly/react-core/dist/styles/base.css'
-import { Page, PageHeader } from '@patternfly/react-core'
+import { Page, PageHeader, PageSection } from '@patternfly/react-core'
+
+import SearchBar from './components/SearchBar'
```
Comment (invincibleJai) [inline src/Homepage.tsx]: css file name should match with component filename

File: src/Homepage.tsx [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 
 import '@patternfly/react-core/dist/styles/base.css'
-import { Page, PageHeader } from '@patternfly/react-core'
+import { Page, PageHeader, PageSection } from '@patternfly/react-core'
+
```
Comment (invincibleJai) [inline src/Homepage.tsx]: why new line?

File: src/components/SearchBar.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { TextInput, Button } from '@patternfly/react-core'
+
+import './SearchBar.css'
+
+interface SearchBarProps {
+  onFormSubmit: () => void;
+}
+
+const SearchBar: React.FC<SearchBarProps> = (props) => {
```
Comment (invincibleJai) [inline src/components/SearchBar.tsx]: can destructure props here

File: src/Homepage.tsx (hunk 3 above)
Comment (rottencandy) [reply src/Homepage.tsx]: Removed

File: src/Homepage.tsx (hunk 2 above)
Comment (rottencandy) [reply src/Homepage.tsx]: Not using any CSS for `Homepage.tsx`. This line is importing the `SearchBar` component. The CSS file for that component is already named `SearchBar.css`

File: src/Homepage.tsx (hunk 1 above)
Comment (rottencandy) [reply src/Homepage.tsx]: Modified.

File: src/components/SearchBar.tsx (hunk 4 above)
Comment (rottencandy) [reply src/components/SearchBar.tsx]: Destructured.

**Pull request merged**